server/planning/content_api/events/__init__.py

```diff
@@ -8,23 +8,39 @@
 # AUTHORS and LICENSE files distributed with this source code, or
 # at https://www.sourcefabric.org/superdesk/license
 
+from typing import Optional
 from superdesk.flask import g
+from pydantic import BaseModel
 from eve.utils import ParsedRequest
 from superdesk.core.module import Module
 from superdesk.core.web import EndpointGroup
 from werkzeug.datastructures import MultiDict
 from superdesk.core.types import Request, Response
 from .event import content_api_event_resource_config, ContentAPIEventService
+from planning.content_api.utils import ALLOWED_PARAMS
 
 event_endpoints = EndpointGroup("events_capi", __name__)
 
 
+class EventListParams(BaseModel):
+    start_date: Optional[str] = None
+    end_date: Optional[str] = None
+    include_fields: Optional[str] = None
+    exclude_fields: Optional[str] = None
+    max_results: Optional[str] = None
+    page: Optional[str] = None
+    where: Optional[str] = None
+    q: Optional[str] = None
+    default_operator: Optional[str] = None
+
+
 @event_endpoints.endpoint("event", methods=["GET"])
-async def get_event_list(args, params, request: Request) -> Response:
+async def get_event_list(args: None, params: EventListParams, request: Request) -> Response:
     service = ContentAPIEventService()
     req = ParsedRequest()
-    req.args = MultiDict()
-
+    req.args = MultiDict(
+        {param: getattr(params, param) for param in ALLOWED_PARAMS if getattr(params, param) is not None}
```

Comment:
You need to provide a default for `getattr` otherwise it will raise an AttributeError